added files for new course (#4758)
Qiskit/documentation: learning/courses/designing-and-leading-quantum-projects/introduction.mdx

---
title: Introduction
description: An introduction to a course designed to guide teams and organizations early in their launch of quantum initiatives.
---

{/* cspell:ignore bootcamps hackathons */}

## Welcome

Welcome to Designing and Leading Quantum Projects, a course designed inform teams and organizations on how to develop capabilities required for realizing quantum initiatives.

> This course is intended for academic, business and organizational leadership in early stages of building quantum projects. The lessons will cover common developmental stages and best practices identified by successful innovators in the field. Most of the content will be applicable across quantum projects broadly, though a few topics will be specific to a particular type of partnership. For example, grant writing might be more often applicable to academic researchers, though grants for startups and enterprises are also important in this ecosystem.

Reviewer: ```suggestion
This course is intended for academic, business, and organizational leadership in the early stages of building quantum projects. The lessons will cover common developmental stages and best practices identified by successful innovators in the field. Most of the content will be applicable across quantum projects broadly, though a few topics will be specific to a particular type of partnership. For example, grant writing might be more often applicable to academic researchers, though grants for startups and enterprises are also important in this ecosystem.
```

__Enterprise organizations:__ Enterprise organizations are established companies that apply technology to solve real business challenges, optimize operations, and create competitive advantage at scale. In the quantum computing ecosystem, they play a key role by identifying high‑value use cases, funding applied innovation, and piloting solutions that demonstrate measurable impact in real‑world environments.

### Quantum Innovation Centers

**Quantum Innovation Centers (QICs)** serve as elite centers of expertise in the growing and ever-changing quantum ecosystem. Many operate as a hub, an industry or academic leader growing a quantum community by partnering with IBM directly, and serving as a central contact point for other organizations. The QIC itself may be any of the organization types mentioned above (academic institution, large enterprise, etc.).

> The resulting community around an IBM QIC has the potential to meet local needs through a bottom-up approach, to offer a wider variety of quantum computing access plans, and to leverage the community for broader expertise. This can be especially important for smaller organizations and startups. Being an IBM QIC establishes your leadership in the field, builds important industry connections, and it grows and opens up the quantum community. **Collaborating with a QIC could facilitate more competitive applications for governmental and private funding.**

Reviewer: ```suggestion
The resulting community around an IBM QIC has the potential to meet local needs through a bottom-up approach, to offer a wider variety of quantum computing access plans, and to leverage the community for broader expertise. This can be especially important for smaller organizations and startups. As an IBM QIC, you establish your leadership in the field and build important industry connections - and you are contributing to the ongoing growth of the quantum community. **Collaborating with a QIC could facilitate more competitive applications for governmental and private funding.**
```
